Responsive Main Menu #107  And Game Play Instructions #109

kamishettysreeja25 opened the pull request:
@niccokunzmann 
I have made the following changes
1- I have Changed the main_menu.css file Which takes care of re sizing the buttons
2-I have added the image instructions.png file which contains the instructions so that i can reduce the css file of the howtoplay.html
If There is any thing more to do please suggest me :)

niccokunzmann commented on howtoplay.html:
```diff
@@ -5,9 +5,10 @@
     <body>
         <p>
-        <h1>This Page contains the game rules </h1>
+        <img src="images/instructions.png" alt="Mountain View" style="width:35%;height:100%;position:absolute;left:35%;">
```
Can you move the style into the css file?

kamishettysreeja25 commented:
Would U like me to add a css file for the howtoplay.html ? I thought It would need a extra file

niccokunzmann commented:
@kamishettysreeja25 This would be good. But before you do it: Please use Inkscape to create the picture and save it as SVG.

kamishettysreeja25 commented:
For Now I have Changed the howtoplay.html and Added howtoplay.css file. I am working on the svg image for the controls  and if the how to play  information for the game play is fine i would like to continue with them

niccokunzmann commented on style/main_menu.css:
```diff
@@ -25,19 +56,20 @@ li a:hover {
+    background-color: #ffcc7e;
     margin: 0px 400px 0px 400px;
     padding: 0;
     text-align: center;
```
You can create improvement pull-requests any time. This could be three:

1. main menu
2. credits
3. howtoplay

kamishettysreeja25 commented:
@niccokunzmann 
I have changed the png file to svg please review it :) 
If anything more is required please tell me

niccokunzmann commented:
@kamishettysreeja25 The design is fine. I would like to merge it.

Please always consider that people may use smaller screens and test your site for that. Mobile phones have a smaller resolution with more height that width.
![capture](https://cloud.githubusercontent.com/assets/564768/13539962/ec859c1a-e253-11e5-8f02-7906d82ed5b7.PNG)
In this case the border overlaps the image.

kamishettysreeja25 commented:
@niccokunzmann 
I have understood the problem I have made the position of texts  and background absolute . But till I am facing the same problem there is a overlapping .Do You have any better way of doing this ? :)

niccokunzmann commented:
The page would just contain this image at 100% width. That is all.

kamishettysreeja25 commented:
@niccokunzmann  
Now the border doesn't ovarlap with the image . Hope it fixes the problem :)

kamishettysreeja25 commented:
@niccokunzmann  
I have separated the css and html part.

niccokunzmann commented on style/howtoplay.css:
```diff
@@ -0,0 +1,11 @@
+body{ 
+	background-color: #ffcc7e;
```
Do we need the following four lines and what are the for?

	width:35%;
	height:100%%;
	position:absolute;
	left:35%;

kamishettysreeja25 commented:
This is for the image to be positioned   towards right of the left margin by 35% and setting the width and height of the image 

width:35%;
height:100%;
position:absolute;
left:35%;

this configuration is same for main menu also  , so that it looks similar to the main menu .

niccokunzmann commented:
@kamishettysreeja25 
If you make the window smaller, the image gets cut off.
I think we can circumvent this by using

    max-width: 40%;
    max-height: 100%;

What do you think?

kamishettysreeja25 commented:
@niccokunzmann 
the image is still getting cut off ater replacing the widht to max-width and height to max-heigth
![screenshot from 2016-03-06 01 28 23](https://cloud.githubusercontent.com/assets/14835922/13550071/d938159c-e33a-11e5-966c-70307677f7ab.png)

niccokunzmann commented:
@kamishettysreeja25 Do you think it is ok if it is cut off this way?

kamishettysreeja25 commented:
@niccokunzmann 
It will fine, as user may not resize the window to so small size and this works fine in the app .
so i think this will not be any problem :)

niccokunzmann commented:
@kamishettysreeja25 ok :)

kamishettysreeja25 commented:
so we can now close the issue ,  closes #109
